packages/blob-storage-client/src/blob-storage-client.js

```diff
@@ -1,6 +1,7 @@
 import { DefaultAzureCredential } from '@azure/identity';
```

Comment:
this should cover every blob storage request. I don't know how to test it though.
For now, I suppose we could just test it on dev and make sure that it doesn't break anything.